fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/socialmedia/SocialMediaViewModel.kt

```diff
@@ -286,6 +288,11 @@ constructor(
       val token = validPoPToken
       val toSearch = token.publicKey.encoded
       senders.contains(toSearch)
+    } catch (e: InvalidPoPTokenException) {
+      // InvalidPoPTokenException means the user did not attend any roll calls and had no token.
+      // This is not an issue worth displaying the error for every single chirps.
+      // This error will already show once in isOwner.
+      return false
     } catch (e: KeyException) {
       logAndShow(getApplication(), TAG, e, R.string.error_retrieve_own_token)
       false
```

Comment:
To simplify a bit the code clarity you can exploit the fact that InvalidPoPTokenException is a subclass of KeyException, so you can use the catch only once for the most generic KeyException and use an if to logAndShow

```suggestion
    } catch (e: KeyException) {
      if (e !is InvalidPoPTokenException) { 
        logAndShow(getApplication(), TAG, e, R.string.error_retrieve_own_token)
      }
      false
```